src/modules/storage.js

```diff
@@ -0,0 +1,40 @@
+export default class localStorage {
+  static getTasks = () => {
+    let tasks;
+    if (localStorage.getItem('tasks') === null) {
+      tasks = [];
+    } else {
+      tasks = JSON.parse(localStorage.getItem('tasks'));
+    }
+    return tasks;
+  };
```

Comment:
- [ ] I am not able to add the tasks to the to do list. The error `localStorage.getItem is not a function
    at localStorage.getTasks` is displayed in the console. This is because class name `localStorage` conflicts with the global `localStorage` object provided by the browser's JavaScript API. You may consider renaming the class to something else. Kindly fix this error so that the tasks can be added and hence interactive ☺️.
    
![image](https://github.com/Paulinelali/webpack-todo-list/assets/99394374/1ba33bef-4132-4882-b109-3ded27f76995)